### PR TITLE
Re: Revision of writing etc.

### PR DESCRIPTION
**Pull request by jparta**
Updated revision (from #7) with suggestions from @raphendyr. History is now linear.

### CODE REVIEW

File: exercises/personalized_number/config.yaml [hunk 1]
```diff
@@ -29,7 +30,7 @@ max_submissions_before_regeneration: 3
 # configuration for the new Docker container grading sandbox
 # the container must have Python 3 installed
 container:
-  image: apluslms/grade-python:3.6-2.7
+  image: apluslms/grade-python:3.5-2.2
```
Comment (raphendyr) [inline exercises/personalized_number/config.yaml]: This change is reverting the version update from before. I'm pretty sure this should be in the PR.

File: exercises/hello_scala/config.yaml [hunk 2]
```diff
@@ -17,6 +17,6 @@ files:
     name: HelloScala.scala
 
 container:
-  image: apluslms/grade-scala:2.12-2.5
+  image: apluslms/grading-scala:2.12
```
Comment (raphendyr) [inline exercises/hello_scala/config.yaml]: This too, `grading-` prefixed containers are deprecated (most).

**Comment by raphendyr** [comment]
You could have force pushed to update the previous PR.

There are still some changes that are clearly not supposed to be here. E.g. changes reverting container version updates.

File: m01_introduction/02_rst.rst [hunk 3]
```diff
@@ -219,7 +219,7 @@ Tips
 
 When you are only writing learning material as RST which contains text, links
 and MathJax notation, you don't need to start A+ to see the results. The
-compiled material is in the subdirectory ``_build/html``, which you can view
+compiled material is in the subdirectory ``_data/html``, which you can view
```
Comment (raphendyr) [inline m01_introduction/02_rst.rst]: This is not correct. Build destination is still `_build/`.

File: m02_programming_exercises/02_hello_world.rst [hunk 4]
```diff
@@ -6,6 +6,14 @@ submission forms can exist anywhere and as many on one page as required.
 The automatic assessment of a submission is defined in the referenced
 YAML file.
 
+This is the configuration file ``docker-compose.yml``:  
+
+.. include:: ../docker-compose.yml
+  :code: yaml
+
+>>>>>>> 770204a... show current docker-compile.yml file
```
Comment (raphendyr) [inline m02_programming_exercises/02_hello_world.rst]: Leftovers from a merge conflict?

File: m01_introduction/02_rst.rst [hunk 5]
```diff
@@ -139,11 +139,11 @@ The usual workflow for editing a course is the following:
 1. Open your text editor, a terminal and a web browser.
 2. Edit some RST files in your text editor.
 3. Give command ``./docker-compile.sh`` in the terminal.
-4. Give command ``rm -rf _data && ./docker-up.sh`` in the terminal.
+4. Give command ``./docker-up.sh`` in the terminal.
 5. Go to ``http://localhost:8000/`` in the web browser to view A+
    running on your machine.
 6. Examine the changes you made in A+.
-7. Press ``q`` or ``Ctrl+C`` in the terminal.
+7. Press ``Q`` or ``Ctrl+C`` in the terminal.
```
Comment (raphendyr) [inline m01_introduction/02_rst.rst]: New functionality with ESC and `S` could be documented here. Not required for this PR, but you can add if you like.

**Comment by jparta** [comment]
What about now?

File: m01_introduction/02_rst.rst [hunk 6]
```diff
@@ -139,11 +139,12 @@ The usual workflow for editing a course is the following:
...
 2. Edit some RST files in your text editor.
 3. Give command ``./docker-compile.sh`` in the terminal.
-4. Give command ``rm -rf _data && ./docker-up.sh`` in the terminal.
+4. Give command ``./docker-up.sh`` in the terminal.
 5. Go to ``http://localhost:8000/`` in the web browser to view A+
    running on your machine.
 6. Examine the changes you made in A+.
-7. Press ``q`` or ``Ctrl+C`` in the terminal.
+7. Press ``Q`` or ``Ctrl+C`` in the terminal to quit and remove data 
+   or ``S`` or ``ESC`` to quit and keep   data.
```
Comment (raphendyr) [inline m01_introduction/02_rst.rst]: There is some extra whitespace between `keep` and `data`.

**Comment by raphendyr** [comment]
Added last whitespace notice, otherwise it looks really good :)

**Comment by raphendyr** [comment]
You could squash all the wording to a single commit. The docker compose include can be in another, but if it's in the same single commit, that would be fine too.

**Comment by jparta** [comment]
Should be nice now.

**Comment by raphendyr** [comment]
thanks

**Pull request merged**